docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -95,46 +116,6 @@ kubectl apply -f <path to target.yaml>
-section compares the file system representations of the component version referencing the blueprint in the OCI registry
-with the component version embedding the blueprint as a local blob.
-
-#### Component Version with External Resource  
-A file system representation of a component version containing the resource through an external reference is shown
-[here](https://github.com/gardener/landscaper/tree/master/docs/guided-tour/blueprints/external-blueprint/component-archive/v2-external). The corresponding component-descriptor describing that component version is
-stored as a top-level file. The component descriptor contains only a single _resource_ with the `name: blueprint`. This
-resource has an _access_ of `type: ociArtifact` that contains a reference to the previously uploaded image.  Besides the
-component-descriptor, there is a directory called blobs at the top-level. This is where the local blobs of a embedded
-resource would be located. Since the blueprint is the only resource of this component version and it is contained
-through an external reference, this directory is empty.
-
-#### Component Version with Local Resource  
-A file system representation of a component version containing the resource as a local blob is shown
-[here](https://github.com/gardener/landscaper/tree/master/docs/guided-tour/blueprints/external-blueprint/component-archive/v2-local).  Again, the corresponding component-descriptor describing that component version
-is stored as a top-level file. Exactly as before, the component descriptor contains only a single _resource_ with the
-`name: blueprint`. But now, this resource has an _access_ of `type: localBlob`. Instead of an `imageReference`, this
-_access_ of `type: localBlob` has a `localReference`. This is the sha256 hash value of the blueprint. If you open the
-blob directory here, you will see that it contains a file with exactly that name. Furthermore, this _access_ has a field
-`mediaType`, which provides information about the format in which the blueprint is stored, here that it is archived
-and compressed (tar+gzip). The _access_ of `type: ociArtifact` did not need to provide this information since the
-`ociArtifact` format is determined through the oci standard and the format of the contents of the oci artifact is
-described within the artifact itself.
-
-These file system representations of component versions can then be uploaded to an OCI registry.
-
-Again, the commands used to create and upload the above component versions can be found
-[here](./commands/upload-component-version.sh).  
-If you want to inspect the uploaded component version (e.g. to find out 
-whether we really have uploaded the one with the external resource), you can do so using the following command:
-
-```shell 
-ocm download componentversion \
-  eu.gcr.io/gardener-project/landscaper/examples//github.com/gardener/landscaper-examples/guided-tour/external-blueprint:2.0.0 \
-  -O component-archive 
-```
```

Comment:
Also this information introduced by you some time ago was quite good. Just my opinion :-(